aa-update-browser: add page (#16882)
tldr-pages/tldr: pages/linux/aa-update-browser.md

# aa-update-browser

> Update AppArmor browser profiles to use supported abstractions.
> Part of the AppArmor suite.

> > Requires root privileges.

Reviewer: ```suggestion
```
The examples already tell you this by requiring sudo

> > More information: <https://manpages.ubuntu.com/manpages/questing/en/man8/aa-update-browser.8.html>.

Reviewer: ```suggestion
> More information: <https://manned.org/aa-update-browser>.
```

- List available browser abstraction profiles:

`sudo aa-update-browser -l`

> - Show what changes would be made to a profile without applying them (dry-run):

Reviewer: ```suggestion
- Show what changes would be made to a profile without applying them ([d]ry-run):
```
This will help users understand what the d is short for.

> `sudo aa-update-browser -d {{/path/to/profile}}`

Reviewer: ```suggestion
`sudo aa-update-browser -d {{path/to/profile}}`
```
Is an absolute path necessary?

- Update a profile with specific abstractions:

> `sudo aa-update-browser -u {{abstraction1,abstraction2}} {{/path/to/profile}}`

Reviewer: ```suggestion
`sudo aa-update-browser -u {{abstraction1,abstraction2}} {{path/to/profile}}`
```

- Remove all abstractions from a profile:

> `sudo aa-update-browser -u '' {{/path/to/profile}}`

Reviewer: ```suggestion
`sudo aa-update-browser -u '' {{path/to/profile}}`
```
